### ISSUE TITLE
Hourly based to-do list

### ISSUE THREAD

**Issue by Lorenzobattistela** [LukeberryPi/phived#8]
I understand that the purpose of phived is to be as clean as possible, without distractions to get it done.

My suggestion is simply adding the option to prefix the list with hours, like a daily schedule. 
We could develop the idea, I was thinking about something like an option side-by-side with the light/dark mode.

Daily schedules are really useful on planning. Many times I've seen myself blocked on a task of my to-do (many reasons, complexity, procrastination). When I started using hourly-based to-dos, I simply moved on to the next task before procrastinating or overthinking it.

**Comment by LukeberryPi** [comment]
hey, lorenzo! interesting idea.

would you care to plot out these changes in a excalidraw/something more visual? i get the general concept, but would like more details to understand if this is something we may pursue in the future.

e.g.: would the quantity of rows change? how many would there be? how does daily/hourly interact?

thanks for the care, let's talk soon

**Comment by Lorenzobattistela** [comment]
Sure, I'll work on something more visual. The quantity of rows change is an option. I would start with the 'now' hour + 1 and then add 1h each row (maintaining the default number of rows and increasing if user wants more hours)

For example: 
Here is 8:24pm, which means the list would start on 9pm, +1 hour per row with the possibilitie to increase it (but not change default).

I'll work in something more visual and comment here. Thanks for responding, hope the project grows!

**Comment by Lorenzobattistela** [comment]
Hello again. I drawed something on excalidraw

![phived](https://user-images.githubusercontent.com/70359945/226133503-c271160a-16c3-4d79-87fe-c0bf22a6ace2.png)

I'm not a design guy, so it's poorly design. But the idea is to be simple. I think we can even leave out the checkbox, to keep the pattern with light/dark mode.

About the implementation, IDK what are your thoughts. I guess we define it on header and use it in tasks component.
I am not used to `tsx`, i'm personally an AI guy, but if you want to explain I can try to implement and learn!

**Comment by Nick-Gabe** [comment]
Ok, so it would by default add last hour + 1 for additional cells, but the user would be able to change right?
I got it, seems a pretty cool idea

**Comment by LukeberryPi** [comment]
this is a well thought out issue and feature, thanks for drawing that up so clearly!

unfortunately, it strays too far away from the project's core concepts. phived was born as a dead-simple to-do list, intended to provide as little complexity as possible, to avoid users escaping by feeling overwhelmed.

with this is mind, i will be closing this issue with the wontfix tag, signaling that this idea will not be moving forward.

thank you again and hope you can find other ways to contribute!
